src/core/Toast/Toaster.tsx

```diff
@@ -51,22 +38,45 @@ export default class Toaster {
     category: ToastCategory,
     settings?: ToastOptions,
   ) {
-    this.addToast({
-      ...settings,
-      content,
-      category,
-    });
+    const lastId = this.toasts[0]?.id ?? 0;
```

Comment:
New toast won't always have new id. Idk if it is bad but maybe to avoid random errors e.g. with keys, we should keep static count as it was done previously.

Reply:
> New toast won't always have new id

Isn't that what the next line does? Or are you saying the user could provide any random id?

Reply:
Also, wasn't static count the cause of #57? Or was it something else?

Reply:
Not sure what to do with this one...

Reply:
Added class variable